### PR TITLE
chore: make lint checks pass

### PR DESCRIPTION
**Pull request by balajismaniam**
**Description**

Regenerated metadata
Linted py files
Added flake8 config

### CODE REVIEW

**Comment by balajismaniam** [comment]
@bharathkkb I have used `EXCLUDE_LINT_DIRS` to ignore `src/frontend/public` directory but the lint checks keep failing still. Do you have a suggestion to fix this. Am I missing something? Also, is there a way to pass args to flake8 in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/build/developer-tools/build/scripts/task_helper_functions.sh#L210. I'm using a config file now but just want to make sure.

**Comment by balajismaniam** [comment]
@donmccasland I had to run `make docker_generate_docs` to make lint checks happy. It made a bunch of changes to several metadata files. Did you happen to add these changes manually? Just making sure before merging.

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -44,3 +44,5 @@ jobs:
             echo dev-tools=${REGISTRY}/${IMAGE}:${VERSION} >> "$GITHUB_OUTPUT"
           fi
       - run: docker run --rm -v ${{ github.workspace }}:/workspace ${{ steps.variables.outputs.dev-tools }} /usr/local/bin/test_lint.sh
+        env:
+          EXCLUDE_LINT_DIRS: '\./src/frontend/public'
```
[the remark this answers is not in the document]
Comment (bharathkkb) [reply .github/workflows/lint.yaml:48]: Also opened https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/1632 to globally skip ico files

**Review by bharathkkb — APPROVED** [review]

**Comment by bharathkkb** [comment]
Quick check of generated metadata also looks good. cc @g-awmalik if any concerns

File: .github/workflows/lint.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (apeabody) [reply .github/workflows/lint.yaml:48]: @bharathkkb - It doesn't look like EXCLUDE_LINT_DIRS is used for the whitespace check.  We should add ico here: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/build/developer-tools/build/scripts/task_helper_functions.sh#L235

[the remark this answers is not in the document]
Comment (bharathkkb) [reply .github/workflows/lint.yaml:48]: Ah good catch! Thanks @apeabody

**Pull request merged**